README.md

```diff
@@ -1,7 +1,7 @@
-![Compiles](https://github.com/Layr-Labs/eigenda-proxy/actions/workflows/build.yml/badge.svg)
+[![Compiles](https://github.com/Layr-Labs/eigenda-proxy/actions/workflows/build.yml/badge.svg)](https://github.com/Layr-Labs/eigenda-proxy/actions/workflows/build.yml)
 ![Unit Tests](https://github.com/Layr-Labs/eigenda-proxy/actions/workflows/unit-tests.yml/badge.svg)
```

Comment:
shouldn't this one be updated too?

Reply:
I'm just not sure if this link will be correct
https://github.com/Layr-Labs/eigenda/blob/master/test.sh